utils/rate_limiter.go

```diff
@@ -55,9 +55,13 @@ type ReconfigurableRateLimiter struct {
 
 // NewRateLimiter creates a new ReconfigurableRateLimiter.
 func NewRateLimiter(creditsPerSecond, maxBalance float64) *ReconfigurableRateLimiter {
+	balance := maxBalance
+	if creditsPerSecond == 0 {
```

Comment:
seems like it should be invalid condition to instantiate rate limiter with <=0 replenishment rate

Reply:
Thanks @yurishkuro.
I feel 0 maybe a valid condition which should have the same effect as {const, 0}. 

Supporting zero case will be useful when we want to totally mute the signal from a service endpoint when remotely controlling the sampling from collector using the perOperation strategy. (Setting the probability to 0 is not enough and we have to set the lowerbound to 0 as well).